src/main/java/dk/kb/netarchivesuite/solrwayback/solr/SolrStats.java

```diff
@@ -0,0 +1,92 @@
+package dk.kb.netarchivesuite.solrwayback.solr;
+
+import com.google.gson.Gson;
+import dk.kb.netarchivesuite.solrwayback.properties.PropertiesLoaderWeb;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.solr.client.solrj.SolrQuery;
+import org.apache.solr.client.solrj.response.QueryResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Get stats through the solr <a href="https://solr.apache.org/guide/8_11/the-stats-component.html">stats component</a>.
+ * Methods in this class
+ */
+public class SolrStats {
+    private static final Logger log = LoggerFactory.getLogger(SolrStats.class);
+    public static final List<String> interestingNumericFields =  Arrays.asList("content_length", "crawl_year", "content_text_length", "image_height", "image_width", "image_size");
+    final List<String> interestingTextFields = Arrays.asList("links", "domain", "elements_used", "content_type",
+                                                                "content_language", "links_images", "type");
+    final List<String> otherNumericFields = Arrays.asList("score", "status_code", "source_file_offset", "_version_", "wayback_date");
```

Comment:
These are not currently used. Should they be default fields if none are provided?

Reply:
They were leftovers from earlier experimenting.